lib/taskana-core/src/main/java/pro/taskana/common/internal/ConfigurationMapper.java

```diff
@@ -1,12 +1,23 @@
 package pro.taskana.common.internal;
 
+import static pro.taskana.common.internal.util.SqlProviderUtil.CLOSING_SCRIPT_TAG;
+import static pro.taskana.common.internal.util.SqlProviderUtil.OPENING_SCRIPT_TAG;
+
 import java.util.Map;
 import org.apache.ibatis.annotations.Param;
 import org.apache.ibatis.annotations.Select;
 import org.apache.ibatis.annotations.Update;
 
 public interface ConfigurationMapper {
 
+  @Select(databaseId = "oracle", value =
+      OPENING_SCRIPT_TAG
```

Comment:
This is inconsistent with all other sql statements in this file.
Please adjust by either using this constant everywhere or nowhere. Thanks!

Reply:
i decided to have everywhere OPENING_SCRIPT_TAG and CLOSING_SCRIPT_TAG where needed